### PR TITLE
Implementation of the file renaming feature

### PR DESCRIPTION
**Pull request by Oleh-Stefanovych**
Now each file in the file list has a `Rename` button to rename it.

The `ESLint` result:
image

### CODE REVIEW

**Comment by Oleh-Stefanovych** [comment]
@dkalinin, PTAL.

**Comment by Oleh-Stefanovych** [comment]
@dkalinin, PTAL.

File: app/component/droppable-component/file-list/index.js [hunk 1]
```diff
@@ -188,4 +214,17 @@ export class FileList extends DroppableComponent {
...
+  /**
+   * Adds a callback that will be executed when the file was renamed.
+   *
+   * @param {function(FileInfo, string)} callback - a callback that will be executed when the
+   * file was renamed. Expects arguments: {@link FileInfo} - a target file and {@link string} -
+   * a new name.
+   * @return {FileList} - the current instance for chaining.
+   */
+  onFileRename(callback) {
+    this._onFileRenameCallbck = callback;
```
Comment (dkalinin) [inline app/component/droppable-component/file-list/index.js]: _onFileRenameCallbck -> _onFileRenameCallback

File: app/component/index.js [hunk 2]
```diff
@@ -31,6 +31,10 @@ export class Component {
     if (Boolean(this._isDraggable)) {
       this.element.draggable = this._isDraggable;
     }
+
+    this.element.addEventListener('dblclick', () => {
```
Comment (dkalinin) [inline app/component/index.js]: Let's add event listener when event handler is passed.

**Pull request merged**